shot/src/main/scala/com/karumi/shot/tasks/Tasks.scala

```diff
@@ -42,6 +44,7 @@ class ExecuteScreenshotTests extends ShotTask {
   def executeScreenshotTests(): Unit = {
     val project = getProject
     val recordScreenshots = project.hasProperty("record")
+    val printBase64 = project.hasProperty("printBase64")
```

Comment:
Could you update the project ``README.md`` with some documentation explaining when this param should be used and how?

Reply:
💯 👍

Reply:
💯  👍